eurocodedesign/units/__init__.py

```diff
@@ -0,0 +1,255 @@
+
+    def __lt__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value < other._value
+
+    def __le__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value <= other._value
+
+    def __eq__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value == other._value
+
+    def __ne__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return not (self == other)
+
+    def __ge__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value >= other._value
+
+    def __gt__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value > other._value
+
+    def to(self, prefix: Prefix):
+        """
+        Convert unit prefix to given prefix
+        @param prefix: Prefix,
+        @return: None
+        """
+        if not self._is_prefix_allowed(prefix):
+            raise TypeError(f"Prefix {prefix} not allowed for {type(self)}")
+        self._prefix = prefix
+        return
+
+
+class Seconds(AbstractUnit):
+    _physical_type = PhysicalType.TIME
+
+
+class Meter(AbstractUnit):
+    _physical_type = PhysicalType.LENGTH
+
+
+class Meter_2(AbstractUnit):
+    _power = 2
+    _physical_type = PhysicalType.AREA
+
+
+class Meter_3(AbstractUnit):
+    _power = 3
+    _physical_type = PhysicalType.VOLUME
+
+
+class Meter_4(AbstractUnit):
+    _power = 4
+    _physical_type = PhysicalType.SECOND_MOMENT_OF_AREA
+
+
+class Meter_per_Second(AbstractUnit):
+    _physical_type = PhysicalType.SPEED
+
+
+class Meter_per_Second_2(AbstractUnit):
+    _physical_type = PhysicalType.ACCELERATION
+
+
+class Kilogram(AbstractUnit):
+    _physical_type = PhysicalType.MASS
+
+
+class Newton(AbstractUnit):
+    _physical_type = PhysicalType.FORCE
+
+
+class Pascal(AbstractUnit):
+    _physical_type = PhysicalType.PRESSURE
+
+
+class Joule(AbstractUnit):
+    _physical_type = PhysicalType.ENERGY
+
+
+class Kelvin(AbstractUnit):
+    _physical_type = PhysicalType.TEMPERATURE
+
+
+_allowed_multiplications = {
+    (Meter, Meter): Meter_2,
+    (Meter_2, Meter): Meter_3,
+    (Meter_2, Meter_2): Meter_4,
+    (Meter_3, Meter): Meter_4,
+    (Meter_per_Second, Seconds): Meter,
+    (Meter_per_Second_2, Seconds): Meter_per_Second,
+    (Kilogram, Meter_per_Second_2): Newton,
+    (Pascal, Meter_2): Newton,
+    (Newton, Meter): Joule,
```

Comment:
This is another example of different physical quantities using the same units. Bending Moment is Nm (the same as Joule). Am I correct in thinking that the current implementation would result in our bending strenghs being of type `Joule`?

Reply:
You're correct. I will add the type Newtonmeter with an Typealias for Nm, and I think even the alias kNm.

What do you think about Multiplication, should addition of 1 Nm +3 J be allowed? I would say yes. Maybe, I will just add a TypeAlias "Nm" for "J".